sdk/src/cameras/ad-fxtof1-ebz/camera_fxtof1.cpp

```diff
@@ -361,6 +361,11 @@ aditof::Status CameraFxTof1::requestFrame(aditof::Frame *frame,
     uint16_t *frameDataLocation;
     frame->getData(FrameDataType::FULL_DATA, &frameDataLocation);
 
+    if (frame == nullptr) {
```

Comment:
the validation should be done **before** the usage of the pointer. `frame` is used before his validation thus segmentation fault will occur if the pointer is NULL making the validation useless.

Reply:
Fixed